modules/prometheus-cpp/1.2.4/MODULE.bazel

```diff
@@ -0,0 +1,19 @@
+module(
+    name = "prometheus-cpp",
+    version = "1.2.4",
+    repo_name = "com_github_jupp0r_prometheus_cpp",
```

Comment:
@meteorcloudy , Shall this be removed ?

Reply:
Yes, but it can be done in the next version. It doesn't really hurt anything, but could be a bit confusing.